extensions/amp-base-carousel/1.0/scroller.js

```diff
@@ -114,6 +115,7 @@ function ScrollerWithRef(
       advance,
       next: () => advance(advanceCount),
       prev: () => advance(-advanceCount),
+      node: containerRef.current,
```

Comment:
Same as the case we had before. We need to make this return the live value. So perhaps we should do this?

```
{
  ...
  get node() {return containerRef.current;},
}
```
?

Reply:
Done.